src/runtime/PromiseStepFunctionObject.cpp

```diff
@@ -0,0 +1,26 @@
+#include "Escargot.h"
+#include "PromiseStepFunctionObject.h"
+
+namespace Escargot {
+
+void* PromiseStepFunctionObject::operator new(size_t size)
+{
+    static bool typeInited = false;
+    static GC_descr descr;
+    if (!typeInited) {
+        GC_word obj_bitmap[GC_BITMAP_SIZE(PromiseStepFunctionObject)] = { 0 };
+        GC_set_bit(obj_bitmap, GC_WORD_OFFSET(PromiseStepFunctionObject, m_onFinally));
```

Comment:
If I'm not mistaken the inherited properties from the parent should also be present is this descriptor. @clover2123 ?

Reply:
@rerobika Yes right. `new` descriptor should also present all GC members of parents.